### DISCUSSION TITLE
What is the process for contributing an entire new scheduler to pytest-xdist?

### DISCUSSION THREAD

**Discussion by vitaly-krugl** [pytest-dev/pytest-xdist#1018] [General]
Dear all, I see that xdist is licensed under the MIT license. However, I didn't find any discussion about contributor licenses. The contribution would be coming through my employer. What is the process for contributing?

**Comment by nicoddemus** [comment]
Hi @vitaly-krugl,

I believe in order to contribute to the pytest-xdist repository, your contribution would have to follow the same license.

**Reply by vitaly-krugl** [reply]
Thank you. Will do.

**Reply by zhubo1234** [reply]
Hello, may I ask a question? I want to write a new scheduler because the existing scheduer does not meet my requirements, but how should I debug it？
